rfcs/rfcOVID-0018.md

```diff
@@ -0,0 +1,104 @@
+# A feature to automatically "yield true" at the end of a file
+
+## Preamble
+
+    Author:  Curtis "Ovid" Poe <curtis.poe@gmail.com>
+    Sponsor:
+    ID:      OVID-0018
+    Status:  Draft
+
+## Abstract
+
+This RFC proposes a `yield_true` feature. When used, the current Perl _file_
+containing `use feature 'yield_true';` will automatically return a true value
+after successful compilation, eliminating the need for a "1" (or other true
+value) at the end of the file.
+
+## Motivation
+
+Eliminate the need for a true value at the end of a Perl file.
+
+## Rationale
+
+There's no need to have a true value be hard-coded in our files that we
+`use`. Further, newer programmers can get confused because sometimes code
+_doesn't_ end with a true value but nonetheless compiles just fine because
+_something_ in the code returned a true value and the code compiles as a
+side-effect.
+
+## Specification
+
+    use feature 'yield_true';
+
+Code using the above does not need to return a magic true value when compiled.
+
+If the module explicitly returns a false value, module loading will fail as it
+does now. If the module author wants the module to fail to load under certain
+conditions, they should die with an appropriate error message rather than
```

Comment:
I'd suggest a slight wording change, to avoid suggesting that the author *themselves* should die ;)

Maybe something to the effect that "If the module author wants ..., they should use the `die` keyword or similar to arrange an appropriate exception to be thrown ..."